src/meta/processors/schemaMan/AlterEdgeProcessor.cpp

```diff
@@ -88,11 +91,15 @@ void AlterEdgeProcessor::process(const cpp2::AlterEdgeReq& req) {
     }
     schema.set_columns(std::move(columns));
 
-    std::vector<kvstore::KV> data;
     LOG(INFO) << "Alter edge " << req.get_edge_name() << ", edgeType " << edgeType;
     data.emplace_back(MetaServiceUtils::schemaEdgeKey(spaceId, edgeType, version),
                       MetaServiceUtils::schemaEdgeVal(req.get_edge_name(), schema));
     resp_.set_id(to(edgeType, EntryType::EDGE));
+    // Now we get default value from meta instead of cache
+    // so don't update
+    if (multiRemove(removeDefaultKeys) != kvstore::SUCCEEDED) {
```

Comment:
modify to  `if (removeDefaultKeys.empty() > 0 && (multiRemove(removeDefaultKeys) != kvstore::SUCCEEDED))`